jsonparser.cpp

```diff
@@ -29,23 +30,36 @@
 #include <QVersionNumber>
 #include <algorithm>
 
-bool compareVersion(const QVariantMap &imageMap1, const QVariantMap &imageMap2)
```

Comment:
please keep the old version (`&` near the variable) to avoid this change being listed